src/rules/function-scale-color-use-always/README.md

```diff
@@ -0,0 +1,83 @@
+# function-scale-color-use-always
```

Comment:
This name feels a bit strange, and is in a format that is not used in stylelint.

I'm not sure what would be a good name that would match stylelint's naming conventions.

Reply:
`scale-color` is meant to change a color by a relative amount, rather than a fixed amount.

What if the rule was called something like: `function-scale-color-relative`. It makes it clear it applies to the scale-color function and that the effect is to make sure it is used to make color changes relative?

Reply:
Hmm... or maybe even more generic `function-color-relative` or `function-no-fixed-color`.

...to make it match a little bit the naming style of the rules in stylelint: `function-name-case`, `function-parentheses-newline-inside`, `property-no-vendor-prefix`, etc.

Reply:
I think `function-color-relative` would be a good name. Made the name change and sent it up.

Reply:
Looks good now!